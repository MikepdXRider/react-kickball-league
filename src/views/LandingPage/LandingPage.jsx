import React from 'react'
import { Link } from 'react-router-dom'

function LandingPage() {
    return (
        <section>
            <h1>Welcome to the kickball league!</h1>
            <Link to='/teams'>Teams List!</Link>
        </section>
    )
}

export default LandingPage
