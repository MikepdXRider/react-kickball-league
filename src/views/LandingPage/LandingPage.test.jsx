import { screen } from "@testing-library/react";
import { render } from "@testing-library/react";
import { MemoryRouter } from "react-router";
import LandingPage from "./LandingPage.jsx";

it ('renders PlayerCard', () => {
    const { container } = render(
    <MemoryRouter>
        <LandingPage />
    </MemoryRouter>
     )
    const title = screen.getByText('Welcome to the kickball league!');
    // or...
    // const title = screen.getByText(/welcome to the kickball league/i);
    

    expect(title).toBeInTheDocument();
    expect(container).toMatchSnapshot();
})