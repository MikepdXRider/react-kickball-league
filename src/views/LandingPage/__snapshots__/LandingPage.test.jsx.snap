// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders PlayerCard page 1`] = `
<div>
  <section>
    <h1>
      Welcome to the kickball league!
    </h1>
    <a
      href="/teams"
    >
      Teams List!
    </a>
  </section>
</div>
`;
